src/lib.rs

```diff
@@ -72,12 +72,15 @@ pub struct GlContext {
 
 impl GlContext {
     pub fn create(
-        parent: &impl HasRawWindowHandle,
```

Comment:
Passing a &dyn instead of a &impl is an opinionated change. I made it mostly because I take &dyn instead of &impl in rafx. I assume passing an &impl T instead of &dyn T is syntactic sugar for making the function:

fn create<T: HasRawWindowHandle>(parent: &T)

I generally favor dyn because I try to avoid generics unless there's a compelling reason to use one, but this is obviously an opinion and I don't think there's a wrong answer here. It would likely require me to change everywhere in rafx that uses &dyn HasRawWindowHandle to &impl HasRawWindowHandle though. Do you have any thoughts/opinions on this?

Reply:
`dyn Trait` implements `Trait`, so if you change the parameter to receive `&(impl HasRawWindowHandle + ?Sized)` you can have both. [A example in playground](https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&code=trait%20MyTrait%20%7B%7D%0Astruct%20A%3B%0Aimpl%20MyTrait%20for%20A%20%7B%7D%0A%0Afn%20dynamic_trait(a%3A%20%26dyn%20MyTrait)%20%7B%0A%20%20%20%20static_trait(a)%3B%0A%7D%0A%0Afn%20static_trait(a%3A%20%26(impl%20MyTrait%20%2B%20%3FSized))%20%7B%0A%20%20%20%20let%20_%20%3D%20a%3B%0A%20%20%20%20println!(%22works%22)%3B%0A%7D%0A%0Afn%20main()%20%7B%0A%20%20%20%20dynamic_trait(%26A)%3B%0A%7D).